package.json

```diff
@@ -28,25 +28,25 @@
     "base64-js": "^1.5.1"
   },
   "peerDependencies": {
-    "@itwin/core-bentley": "4.11.1",
-    "@itwin/core-common": "4.11.1",
-    "@itwin/core-frontend": "4.11.1"
+    "@itwin/core-bentley": "5.0.0",
+    "@itwin/core-common": "5.0.0",
+    "@itwin/core-frontend": "5.0.0"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "4.11.1",
-    "@itwin/core-bentley": "4.11.1",
-    "@itwin/core-common": "4.11.1",
-    "@itwin/core-frontend": "4.11.1",
-    "@itwin/core-geometry": "4.11.1",
-    "@itwin/core-i18n": "4.11.1",
-    "@itwin/core-markup": "4.11.1",
-    "@itwin/core-orbitgt": "4.11.1",
-    "@itwin/core-quantity": "4.11.1",
-    "@itwin/ecschema-metadata": "4.11.1",
+    "@itwin/appui-abstract": "5.0.0",
+    "@itwin/core-bentley": "5.0.0",
+    "@itwin/core-common": "5.0.0",
+    "@itwin/core-frontend": "5.0.0",
+    "@itwin/core-geometry": "5.0.0",
+    "@itwin/core-i18n": "5.0.0",
+    "@itwin/core-markup": "5.0.0",
+    "@itwin/core-orbitgt": "5.0.0",
+    "@itwin/core-quantity": "5.0.0",
+    "@itwin/ecschema-metadata": "5.0.0",
     "@itwin/eslint-plugin": "4.0.0-dev.48",
```

Comment:
can we move off the dev release here

Reply:
The last time I tried a newer eslint plugin, it wasn't possible to make it work at all with the updated config style. I will investigate again, though.

Reply:
Done.